app/config/ApplicationConfig.scala

```diff
@@ -16,46 +16,61 @@
 
 package config
 
-import play.api.Mode.Mode
-import play.api.Play._
-import play.api.{Configuration, Play}
-import uk.gov.hmrc.play.config.ServicesConfig
-
-trait ApplicationConfig {
-
-  val defaultBetaFeedbackUrl: String
-  def betaFeedbackUrl(returnUri: String): String
-  val betaFeedbackUnauthenticatedUrl: String
-  val analyticsToken: Option[String]
-  val analyticsHost: String
-  val reportAProblemPartialUrl: String
-  val reportAProblemNonJSUrl: String
-  val defaultTimeoutSeconds: Int
-  val timeoutCountdown: Int
-}
+import javax.inject.Inject
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import utils.AtedSubscriptionUtilsImpl
+
+import scala.util.Try
+
+class ApplicationConfig @Inject()(val servicesConfig: ServicesConfig,
+                                  val atedSubsUtils: AtedSubscriptionUtilsImpl) {
```

Comment:
Not sure this needs to be here - if for some reason it is, it shouldn't be the `Impl` that is referenced here.